tests/unit/executors/encoders/test_numeric.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+
+from jina.executors.encoders.numeric import TransformEncoder
+
+input_dim = 5
+target_output_dim = 5
+
+
+@pytest.fixture
+def model():
+    class SimpleModel:
+
+        def fit(self, data):
+            return data
+
+        def transform(self, data):
+            return data
+
+    simple_model = SimpleModel()
+    return simple_model
+
+
+def test_test_transformencoder(model):
+    train_data = np.random.rand(100, input_dim)
+    encoder = TransformEncoder(output_dim=target_output_dim)
+    encoder.model = model
+    encoder.train(train_data)
+
+    test_data = np.random.rand(10, input_dim)
+    encoded_data = encoder.encode(test_data)
+    assert encoded_data.shape == (test_data.shape[0], target_output_dim)
+    assert type(encoded_data) == np.ndarray
```

Comment:
i would do assert isinstance(encoded_data, np.ndarray)

Reply:
encode() will use the trained model. Can we do that in separate tests?

Reply:
It makes sense to cover more. I will try it